client/my-sites/upgrades/domain-management/dns/dns-list.jsx

```diff
@@ -38,7 +34,7 @@ const DnsList = React.createClass( {
 				<DnsRecord
 					key={ index }
 					dnsRecord={ dnsRecord }
-					deleteDns={ this.deleteDns }
+					deleteDns={ this.handleDeleteDns }
```

Comment:
@stephanethomas is right. There was long discussion how to name methods. In the end we decided to use method names that better reveal intention :) So, yes, we should keep `deleteDns`.